src/main/flight/failsafe.c

```diff
@@ -63,14 +63,14 @@ static failsafeState_t failsafeState;
 PG_REGISTER_WITH_RESET_TEMPLATE(failsafeConfig_t, failsafeConfig, PG_FAILSAFE_CONFIG, 2);
 
 PG_RESET_TEMPLATE(failsafeConfig_t, failsafeConfig,
-    .failsafe_throttle = 1000,                       // default throttle off.
-    .failsafe_throttle_low_delay = 100,              // default throttle low delay for "just disarm" on failsafe condition
-    .failsafe_delay = 10,                                // 1 sec, can regain control on signal recovery, at idle in drop mode
+    .failsafe_throttle = 1000,                           // default throttle off.
+    .failsafe_throttle_low_delay = 100,                  // default throttle low delay for "just disarm" on failsafe condition
+    .failsafe_delay = 15,                                // 1 sec stage 1 period, can regain control on signal recovery, at idle in drop mode
```

Comment:
is 15 still equals to 1 second, previous value was 10 for 1 sec. misleading comment?

Reply:
Thanks, I'll fix that.